sources/api/apiserver/src/server/mod.rs

```diff
@@ -352,6 +353,27 @@ async fn get_configuration_files(
     Ok(ConfigurationFilesResponse(resp))
 }
 
+/// Reboots the machine
+async fn reboot() -> Result<HttpResponse> {
+    debug!("Rebooting in 10 seconds");
+    // Since there's no logind support, '/sbin/shutdown' cannot schedule reboot and will immediately reboot if called directly.
+    // There is a chance that we won't be able to return a response code in time before being terminated.
+    // Currently relying on systemd to add a relay before the reboot.
+    ensure!(
+        Command::new("/bin/systemd-run")
+            // Add a 10 second delay to shutdown
+            .arg("--on-active=10")
+            .arg("shutdown")
+            .arg("-r")
+            .status()
+            .context(error::SystemdRun)?
+            .success(),
+        error::Reboot
```

Comment:
There are a lot of reasons this oculd happen, so I think we should include the stderr output and return code if the command fails, which would mean using `output` instead of `success` and the simple `ensure`.